sdk/resourcemanager/azure-resourcemanager-containerregistry/src/main/java/com/azure/resourcemanager/containerregistry/implementation/RegistriesWebhooksClientImpl.java

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.resourcemanager.containerregistry.implementation;
```

Comment:
This file is previously `WebhooksClientImpl.java`, which is overwritten by generated file with same name. So rename it as `RegistriesWebhooksClientImpl.java`.